src/MCMCChains.jl

```diff
@@ -12,11 +12,27 @@ using Formatting
 import StatsBase: autocov, counts, sem, AbstractWeights,
     autocor, describe, quantile, sample, summarystats, cov
 using Requires
+using Tables
+using TableTraits
+using IteratorInterfaceExtensions
```

Comment:
Since you always write Tables.rows etc explicitly (which I like a lot BTW) you should just change it to
```suggestion
import Tables
import TableTraits
import IteratorInterfaceExtensions
```